pkg/component/worker/kernelsetup_linux.go

```diff
@@ -43,9 +43,9 @@ func hasFilesystem(filesystem string) bool {
 }
 
 func modprobe(module string) {
-	err := exec.Command("modprobe", module)
+	out, err := exec.Command("modprobe", module).CombinedOutput()
 	if err != nil {
-		logrus.Warnf("failed to load %s kernel module: %s", module, err)
+		logrus.Warnf("failed to load %s kernel module: %s, out=%q", module, err.Error(), string(out))
```

Comment:
No need to cast a byte array to string when using %s or %q in the format string.

```suggestion
		logrus.WithError(err).Warnf("failed to load kernel module %s: %s, module, out)
```